src/app/hearing-part/utils/hearing-utils.ts

```diff
@@ -0,0 +1,9 @@
+import { ListingCreate } from '../models/listing-create';
+
+export function isMultiSessionListing(listing: ListingCreate): boolean {
+    if (listing && listing.hearing) {
+        return listing.hearing.duration.asDays() >= 1;
```

Comment:
Shouldn't it be 'greater than': listing.hearing.duration.asDays() > 1; ?

Reply:
>= because minimum for multiple session is 1 day.